doc/api/http.markdown

```diff
@@ -966,6 +966,8 @@ Example:
 
     http.get("http://www.google.com/index.html", function(res) {
       console.log("Got response: " + res.statusCode);
+      // consume response body
+      res.resume();
```

Comment:
So I dug a bit further into this. The response in the callback is an instance of ```http.IncomingMessage```... and finding that out was not exactly obvious. ```http.IncomingMessage``` implements the Readable stream interface, which is where it inherits ```.resume``` from.

Quite a bit of digging is necessary to find out what this function does. While it may not make sense to duplicate documentation, perhaps we can find a better way to discover inherited properties

Any thoughts @nodejs/documentation?

If there is an interest in this I'll move it to a separate issue to avoid derailing this PR.